src/hot_reload.cljs

```diff
@@ -1,12 +1,9 @@
-(ns status-im.reloader
+(ns hot-reload
```

Comment:
I think this namespace could live inside the `src/status-im/setup/` directory. From the docs:

> Contains namespaces that are mostly used to initialize the application, configure test runners, etc. In general, such namespaces should not be required from the outside.

 Although this namespace is required in some places, I see it as boring plumbing not worthy of living at the top level directory.